skiko/src/commonTest/kotlin/org/jetbrains/skia/PathTests.kt

```diff
@@ -213,11 +202,11 @@ class PathTests {
         assertTrue(Path.isQuadDegenerate(Point(0f, 0f), Point(0f, 0f), Point(0f, 0f), false))
         assertFalse(Path.isCubicDegenerate(Point(0f, 0f), Point(10f, 0f), Point(0f, 0f), Point(0f, 0f), false))
         assertTrue(Path.isCubicDegenerate(Point(0f, 0f), Point(0f, 0f), Point(0f, 0f), Point(0f, 0f), false))
-        assertContentEquals(
+        assertContentCloseEnough(
             arrayOf(Point(0f, 20f), Point(6.666667f, 13.333334f)),
             Path.convertConicToQuads(Point(0f, 20f), Point(20f, 0f), Point(40f, 20f), 0.5f, 1)
         )
-        assertContentEquals(
+        assertContentCloseEnough(
             arrayOf(
```

Comment:
Just to be sure - what exactly changed so that we need to relax out equality assertions? 
May be it worth (not in this PR I mean - in general) to have a platform-specific assertions that are strictly equal wherever possible and relaxed on platform we know we can not guarantee it?

Reply:
This is made for JS purposes: in JS code "Float"s have greater precision, so strict tests fail